src/core/fetchers/helpers.ts

```diff
@@ -38,4 +39,29 @@ export const createAuthHeader = (
       }
     : {};
 
+// Creates a "by author, author and author"-string
+// String interpolation todo?
+export const getContributors = (creators: string[]) => {
+  let returnContentString = "";
+
+  // Todo this is sortof a hack, but using t: UseTextFunction as argument
+  // makes the components re-render.
+  const {
+    text: { data: texts }
+  } = store.getState();
+
+  if (creators && creators.length > 0) {
+    if (creators.length === 1) {
+      returnContentString = `${texts.materialByAuthorText} ${creators.join(
+        ", "
+      )}`;
+    } else {
+      returnContentString = `${texts.materialByAuthorText} ${creators
+        .slice(0, -1)
+        .join(", ")} ${texts.materialAndAuthorText} ${creators.slice(-1)}`;
+    }
+  }
+  return returnContentString;
+};
```

Comment:
Could you turn this into a hook instead, so we can use the `useSelector()` hook instead?
I am not fund of including the store directly if we can avoid it.
I have seen it sometimes creating dependency cycles and it would be nice to have it wrapped in some kind of abstraction.

Reply:
I've just moved this function, and I will actually return to this later. I'll add a todo?

Reply:
TODO: Make GitHub Copilot please transform all TODO's into code.
Yes a TODO will DO :)